schedule/yam/agama_unattended_patterns.yaml

```diff
@@ -7,4 +7,5 @@ schedule:
   - yam/agama/agama_auto
   - installation/grub_test
   - installation/first_boot
+  - yam/validate/validate_registered_addons
```

Comment:
file name doesn't make sense anymore and `unattended` should be at the end.

Reply:
updated